### ISSUE TITLE
Importing AMK file generated in iOS does not show patient in macOS.

### ISSUE THREAD

**Issue by zdavatz** [zdavatz/AmiKo-iOS#40]
1. Import AMK File into AmiKo for macOS. 
2. Patient Data is "Null" and is not being imported.

**Comment by bettar** [comment]
Maybe as the AMK file is sent through email, the base64 stream is broken down into lines by inserting newline characters. The email client would do that. The extra character would make the decoding fail on AmiKo macOS. Just a guess.

**Comment by bettar** [comment]
Actually I have just tried that and it solves the problem

**Comment by zdavatz** [comment]
Well then fix it in AmiKo for iOS please.

**Comment by zdavatz** [comment]
1. Create a prescription in the simulator with doctor, patient and two packages.
2. Import it into AmiKo for macOS.
3. Same issue. The patient is NULL.

**Comment by zdavatz** [comment]
<img width="349" alt="bildschirmfoto 2018-04-05 um 17 53 00" src="https://user-images.githubusercontent.com/4953/38377033-3b3eff82-38fa-11e8-9122-b50bbe557be5.png">

**Comment by zdavatz** [comment]
`file` Command shows the following as well:
**File created with AmiKo macOS**
```
$ file RZ_2018-04-05T174356.amk
RZ_2018-04-05T174356.amk: ASCII text, with CRLF line terminators
```
**File crated with AmiKo iOS**
```
$ file RZ_2018-03-23T115542.amk
RZ_2018-03-23T115542.amk: ASCII text, with very long lines, with no line terminators
```

**Comment by bettar** [comment]
AmiKo iOS was breaking down the base-64 stream in lines, because of the option NSDataBase64Encoding64CharacterLineLength, now removed

**Comment by zdavatz** [comment]
should be solved with commit: 3aa3d941c1e44117769
